Symbology/web/VBET/vbet_active_channel.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$schema": "https://xml.riverscapes.xyz/Symbology/web/vector.schema.json",
+    "legend": [
+      ["hsl(206, 70%, 72%)", "Estimated Active Channel"]
+    ],
+    "layerStyles": [
+        {
+            "id": "vbet-active-channel-cd8gcb",
+            "type": "fill",
+            "source": "composite",
+            "source-layer": "vbet_active_channel-cd8gcb",
+            "layout": {},
+            "paint": {"fill-color": "hsl(206, 70%, 72%)", "fill-opacity": 0.7}
+        }
```

Comment:
@shelbysawyer  The issue is that the square bracket doesn't terminate.

Once you get the brackets in the right order this shouldpass.